drivers/eaton-pdu-marlin-mib.c

```diff
@@ -876,50 +849,15 @@ static snmp_info_t eaton_marlin_mib[] = {
 		".1.3.6.1.4.1.534.6.6.7.5.1.1.4.%i.%i",
 		NULL, SU_FLAG_STATIC | SU_OUTLET_GROUP | SU_TYPE_DAISY_1,
 		&marlin_outlet_group_type_info[0], NULL },
-#if WITH_SNMP_LKP_FUN
-	/* Phase to which an outlet-group is connected
-	 * (Caution: the value is numeric, and need to append "L" -- TODO)
-	 * groupPhaseID:  // FIXME-Check on real device
+	/* Phase to which an outlet-group is connected:
+	 * We use the following OID, which gives the voltage measurement type
+	 * groupVoltageMeasType.0.1; Value (Integer): singlePhase  (1)
 	 */
 	/* FIXME: RFC on key name is needed when backporting to NUT upstream ; check type (number? string?) and flags (daisy?) */
 	{ "outlet.group.%i.phase", 0, SU_INFOSIZE,
-		".1.3.6.1.4.1.534.6.6.7.5.1.1.10.%i.%i",
+		".1.3.6.1.4.1.534.6.6.7.5.3.1.2.%i.%i",
```

Comment:
Do we still have two or three OID entries of different preference to pick from, or just one (applicable for all firmware versions, including legacy) now?

Reply:
just one now